micro_dl/cli/image_inference.py

```diff
@@ -106,8 +115,25 @@ def run_prediction(args, gpu_ids, gpu_mem_frac):
     with open(config_name, 'r') as f:
         config = yaml.load(f)
     # Load frames metadata and determine indices
-    frames_meta = pd.read_csv(os.path.join(args.image_dir, 'frames_meta.csv'))
-    split_idx_name = config['dataset']['split_by_column']
+    network_config = config['network']
+    dataset_config = config['dataset']
+    trainer_config = config['trainer']
+    frames_meta = pd.read_csv(os.path.join(args.image_dir, 'frames_meta.csv'),
+                              index_col=0)
+    test_tile_meta = pd.read_csv(os.path.join(args.model_dir, 'test_metadata.csv'),
```

Comment:
sorry, test_metadata is frames_meta['pos_idx'] in split_idx['test']. Is this different than what we had before? Looks like both approaches will get the same output! what am i missing?

Reply:
got it, to include only the slices used in tiling. :-)

Reply:
Yes! Otherwise all the slices will be used for testing :-)